lib: Remove XXX comment in readline and change to output debuglog

kohei-takata opened the pull request:
refs #4642 
Remove a comment that has a word 'XXX'.
And add a line to output debuglog of error.

yosuke-furukawa commented on lib/readline.js:
```diff
@@ -380,7 +381,7 @@ Interface.prototype._tabComplete = function() {
 
     if (err) {
-      // XXX Log it somewhere?
+      debug(err);
```
Congratulations your 1st contribution. 

You would be better to print more information. e.g. 
```javascript
debug('tab complete error ', err);
```

like https://github.com/nodejs/node/blob/master/lib/repl.js#L230

Fishrock123 commented:
@kohei-takata Could you please prefix the commit message with `readline:` instead of `lib:`?

Also please make sure the commit message (the first line, where the prefix is) is no longer than 50 characters. :)

jasnell commented:
LGTM with the nits @Fishrock123 and @yosuke-furukawa mentioned addressed.

kohei-takata commented:
@yosuke-furukawa 
Thanks :smile: 
I've changed arguments of `debug()`.

@Fishrock123 
Thanks for your review!
I've changed prefix of the commit message and shorten :smile: 

@jasnell 
Thanks!

cjihrig commented:
LGTM pending CI

Fishrock123 commented:
CI: https://ci.nodejs.org/job/node-test-pull-request/1255/

jasnell commented:
Failures in CI look unrelated. Updated commit LGTM

Trott commented:
CI is a little more robust right now then it was 13 hours ago, so let's try CI again:
https://ci.nodejs.org/job/node-test-pull-request/1268/

Trott commented:
CI is green! \o/

jasnell commented:
Great job @Trott. Great to see.
On Jan 15, 2016 9:24 PM, "Rich Trott" <notifications@github.com> wrote:

> CI is green! \o/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodejs/node/pull/4690#issuecomment-172159845>.
>

kohei-takata commented:
@Trott Thanks!

jasnell commented:
Landed in ad63d35
Thank you @kohei-takata !
